README.md

```diff
@@ -89,7 +89,7 @@ class UserTableSeeder extends AbstractSeeder
 {
     public function run(): \Generator
     {
-        foreach (UserFactory::new()->times(100)->create() as $user) {
+        foreach (UserFactory::new()->times(100)->make() as $user) {
             yield $user;
         }
     }
```

Comment:
Provide some examples of using new features

Reply:
Readme updated